iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm

```diff
@@ -138,6 +139,9 @@ - (IBAction)login
       catch (exception const & ex)
       {
         LOG(LWARNING, ("Error login", ex.what()));
+        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
```

Comment:
Это уже тебе обсуждать с маркетологами. Разница в регистрации в осме и авторизации вполне может быть им интересна.

> On Mar 25, 2016, at 6:56 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/CustomViews/Login/MWMAuthorizationOSMLoginViewController.mm <https://github.com/mapsme/omim/pull/2511#discussion_r57457291>:
> 
> > @@ -138,6 +139,9 @@ - (IBAction)login
> >        catch (exception const & ex)
> >        {
> >          LOG(LWARNING, ("Error login", ex.what()));
> > +        [Statistics logEvent:@"Editor_Reg_request_result" withParameters:@{kStatIsSuccess : kStatNo,
> Кто мне объяснит отличия в этих ключах статистики в вики? Я предполагаю, что ошиблись при формулировке задачи. Нам нужен факт успеха или ошибка после входа через осм фб или гугл. Дополнительных смыслов вносить в события не вижу.
> … <x-msg://38/#>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2511/files/fb1663fdc02ca41305836057f319abacc56069b7..da074e5c2eb87dea5984b12046370e4a6f55f36b#r57457291>